### PR TITLE
add clia as an identifier in organization

### PR DESCRIPTION
**Pull request by BobanL**
# BACKEND PULL REQUEST

## Related Issue

- Currently, we are sending CLIA in the test event export. We should at least send the same data via FHIR bundle.

## Changes Proposed

- Add CLIA as an identifier on the organization

### CODE REVIEW

File: backend/src/main/java/gov/cdc/usds/simplereport/api/converter/FhirConverter.java [hunk 1]
```diff
@@ -304,6 +305,13 @@ public static Practitioner convertToPractitioner(Provider provider) {
   public static Organization convertToOrganization(Facility facility) {
     var org = new Organization();
     org.setId(facility.getInternalId().toString());
+    org.addIdentifier()
+        .setUse(IdentifierUse.OFFICIAL)
+        .setValue(facility.getCliaNumber())
+        .getType()
+        .addCoding()
+        .setSystem("http://terminology.hl7.org/CodeSystem/v2-0301")
```
Comment (zdeveloper) [inline backend/src/main/java/gov/cdc/usds/simplereport/api/converter/FhirConverter.java:313]: Should this be in the `FhirConstants.java` as universalIdTypeSystem?

**Review by zdeveloper — CHANGES_REQUESTED** [review]
LGTM! minor question

**Review by zdeveloper — APPROVED** [review]
LGTM!

**Review by johanna-skylight — APPROVED** [review]
LGTM

**Pull request merged**